bitcoinlib/data/providers.json

```diff
@@ -306,5 +306,71 @@
     "priority": 10,
     "denominator": 100000000,
     "network_overrides": null
+  },
+  "groestlcoinesplora": {
+    "provider": "groestlcoinesplora",
+    "network": "groestlcoin",
+    "client_class": "GroestlcoinEsploraClient",
+    "provider_coin_id": "",
+    "url": "https://esplora.groestlcoin.org/api/",
+    "api_key": "",
+    "priority": 10,
+    "denominator": 100000000,
+    "network_overrides": null
+  },
+  "groestlcoinesplora.testnet": {
+    "provider": "groestlcoinesplora",
+    "network": "groestlcoin_testnet",
+    "client_class": "GroestlcoinEsploraClient",
+    "provider_coin_id": "",
+    "url": "https://esplora.groestlcoin.org/testnet/api/",
+    "api_key": "",
+    "priority": 10,
+    "denominator": 100000000,
+    "network_overrides": null
+  },
+  "groestlsight": {
+    "provider": "groestlsight",
+    "network": "groestlcoin",
+    "client_class": "GroestlsightClient",
+    "provider_coin_id": "",
+    "url": "https://groestlsight.groestlcoin.org/api/",
+    "api_key": "",
+    "priority": 10,
+    "denominator": 100000000,
+    "network_overrides": null
+  },
+  "groestlsight.testnet": {
+    "provider": "groestlsight",
+    "network": "groestlcoin_testnet",
+    "client_class": "GroestlsightClient",
+    "provider_coin_id": "",
+    "url": "https://groestlsight-test.groestlcoin.org/api/",
+    "api_key": "",
+    "priority": 10,
+    "denominator": 100000000,
+    "network_overrides": null
+  },
+  "cryptoid.groestlcoin": {
+    "provider": "cryptoid",
+    "network": "groestlcoin",
+    "client_class": "CryptoID",
+    "provider_coin_id": "grs",
+    "url": "https://chainz.cryptoid.info/",
+    "api_key": "",
+    "priority": 10,
+    "denominator": 100000000,
+    "network_overrides": null
+  },
+  "cryptoid.groestlcoin.testnet": {
+    "provider": "cryptoid",
+    "network": "groestlcoin_testnet",
+    "client_class": "CryptoID",
+    "provider_coin_id": "grs-test",
+    "url": "https://chainz.cryptoid.info/",
+    "api_key": "",
+    "priority": 10,
+    "denominator": 100000000,
+    "network_overrides": null
   }
 }
```

Comment:
},
  "blockchair.groestlcoin": {
    "provider": "blockchair",
    "network": "groestlcoin",
    "client_class": "BlockChairClient",
    "provider_coin_id": "",
    "url": "https://api.blockchair.com/groestlcoin/",
    "api_key": "",
    "priority": 10,
    "denominator": 100000000,
    "network_overrides": null

Reply:
please add blockchair for grs also in providers.json